Gems/Multiplayer/Code/Include/Multiplayer/Components/SimplePlayerSpawnerComponentBus.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzCore/Outcome/Outcome.h>
```

Comment:
Outcome is not used, remove it to keep compile times down. Remember there's a bit push on getting our compile-times under control right now.

Reply:
Thank you!